project.php

```diff
@@ -1654,6 +1654,10 @@ function echo_download_zip($link_text, $discriminator)
     echo "</a>";
     echo_byte_size($filesize_b);
     echo_last_modified($last_modified);
+    // output a warning if the file was generated before the UTF-8 cutover
+    if ($discriminator == "" && $last_modified && $last_modified < 1589698800) {
```

Comment:
would it make sense to translate the date in a comment?

Reply:
Indeed - added.